examples/java/src/main/java/org/apache/beam/examples/multilang/PythonDataframeWordCount.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.multilang;
+
+import org.apache.beam.examples.common.ExampleUtils;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.extensions.python.PythonExternalTransform;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.metrics.Counter;
+import org.apache.beam.sdk.metrics.Distribution;
+import org.apache.beam.sdk.metrics.Metrics;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.Validation.Required;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.MapElements;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.transforms.SimpleFunction;
+import org.apache.beam.sdk.util.PythonCallableSource;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.Row;
+
+/**
+ * An example that counts words in Shakespeare and utilizes a Python external transform.
+ *
+ * <p>This class, {@link PythonDataframeWordCount}, uses Python DataframeTransform to count words
+ * from the input text file. The Python expansion service provided by --expansionService must allow
+ * the expansion of apache_beam.dataframe.transforms.DataframeTransform (which can be done by
+ * passing --fully_qualified_name_glob commandline option when launching the expansion service).
+ *
+ * <p>Note that, for using Dataflow Runner, you should specify the following two additional
+ * arguments:
+ *
+ * <pre>{@code
+ * --experiments=use_runner_v2
+ * --sdkHarnessContainerImageOverrides=.*python.*,gcr.io/apache-beam-testing/beam-sdk/beam_python3.8_sdk:latest
```

Comment:
In the following examples, I simplified by providing instructions for running with the latest released Beam version. Can we do the same here ? Users should be able to just run this example without additional setup (no need to startup an expansion service or push containers for example).

https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/kafkataxi/README.md
https://github.com/apache/beam/tree/master/examples/multi-language

Reply:
Changed to use auto expansion service from PythonExternalTransform (However, the example will not work until we release a newer SDK version since PythonExternalTransform launches an expansion service by installing a released SDK)